depinject/go.mod

```diff
@@ -3,6 +3,7 @@ module cosmossdk.io/depinject
 go 1.20
 
 require (
+	cosmossdk.io/api v0.7.5
```

Comment:
Depinject shouldn't import api.

Reply:
I guess it was because of this:

```diff
--	_ "cosmossdk.io/depinject/appconfig/v1alpha1"
++	_ "cosmossdk.io/api/cosmos/app/v1alpha1"
```

Not sure what is going on with these proto things, how they can stop working just like that

Reply:
We bumped buf in proto-builder 0.15, so that could be that: contrib/Dockerfile

Reply:
I don't know how to fix this, the internal protos are using `cosmos.app.v1alpha1.module`

Reply:
@julienrbrt fixed, pls re-review